src/types/NodeType.js

```diff
@@ -32,14 +32,12 @@ type ObjStub = {
   id: string,
 };
 
-type NodeTypeConfig<R: Resource<*>> = GraphQLObjectTypeConfig<
-  ObjStub,
-  Context,
-> & {
+type NodeTypeConfig<R: Resource<*>> = {|
+  ...GraphQLObjectTypeConfig<ObjStub, Context>,
```

Comment:
spreading vs &? any difference?

Reply:
v14 makes the config type exact. the union of an exact type with something else is... pointless